.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ env:
   ARM_LOCATION: westus
   ARM_SUBSCRIPTION_ID: ${{ secrets.ARM_SUBSCRIPTION_ID }}
   ARM_TENANT_ID: ${{ secrets.ARM_TENANT_ID }}
-  AWS_ACCESS_KEY_ID: ' ${{ secrets.AWS_ACCESS_KEY_ID }}'
+  AWS_ACCESS_KEY_ID: " ${{ secrets.AWS_ACCESS_KEY_ID }}"
```

Comment:
My editor reformatted from `'` to `"` for quotes, but is the space here intended I the first place? 🤔